python/utils/transactions_processor.py

```diff
@@ -33,6 +38,25 @@ def __init__(
 
         self.init_db_tables()
 
+        # Start the Prometheus client; this is a non-blocking call.
+        start_http_server(self.config.monitoring_port)
+
+        # Start the health server.
+        def start_health_server() -> None:
+            class Handler(http.server.SimpleHTTPRequestHandler):
+                def do_GET(self):
+                    self.send_response(http.HTTPStatus.OK)
+                    self.end_headers()
+                    self.wfile.write(b"OK")
```

Comment:
Can you add more comment explaining what this does? How often do we send this response?

Reply:
done, default is every 10 seconds from kubelet.

Reply:
Thanks! Is this related to the 
```
indexer_grpc_http2_ping_interval_in_secs: 60
indexer_grpc_http2_ping_timeout_in_secs: 10
``` 
config vars that exist in the Rust config? If so we should add these to Python too.

Reply:
oh this is a different config; i can update it here as well.

Reply:
What are those vars for?

Reply:
There are to make sure connection is alive from client side. Without these configs, clients trust the connections blindly, which can be a problem if the underlying network is not stable.

Reply:
done.